README.md

```diff
@@ -5,7 +5,7 @@ Chart.js plugin to display labels on pie, doughnut and polar area chart. Origina
 [Demo](http://emn178.github.io/chartjs-plugin-labels/samples/demo/)
 
 ## Download
-[Compress](https://raw.github.com/emn178/chartjs-plugin-labels/master/build/chartjs-plugin-labels.min.js)  
+[Compress](https://raw.github.com/emn178/chartjs-plugin-labels/master/build/chartjs-plugin-labels.min.js)
```

Comment:
I'd recommend reverting changes to any lines that don't strictly need to be changed. Otherwise it'll be a real pain to merge updates.